apps/molgenis-components/src/docs/forms/InputSearch.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <demo-item id="input-search" label="Search Input (extends BaseInput)">
```

Comment:
I thought you find a way to keep the 'docs' with the component it is documenting. Now we have to maintain two files. I really would like to prevent that.

Reply:
this would mean spending more time on the showCase setup , we discussed this and decided on not doing this